pages/apis/javascript/latest/belt/array.mdx

```diff
@@ -6,6 +6,10 @@ Utililites for `Array` functions.
 
 </Intro>
 
+## Note about index syntax
```

Comment:
level 2 headings are hidden by default... use ### instead, otherwise the header will not be visible

Reply:
Done!